plugin/src/main/java/org/opensearch/ml/indices/MLIndicesHandler.java

```diff
@@ -68,6 +68,7 @@ public void initMLConfigIndex(ActionListener<Boolean> listener) {
     public void initMLIndexIfAbsent(MLIndex index, ActionListener<Boolean> listener) {
         String indexName = index.getIndexName();
         String mapping = index.getMapping();
+        final Map<String, Object> indexSettings = Map.of("index.auto_expand_replicas", "0-5");
```

Comment:
Can we add any comment why we chose `0-5`?

Reply:
To be consistent with AOS default settings